### PR TITLE
new ux - symptom report

### PR DESCRIPTION
**Pull request by mellowgeek**
Updated primary and accent colors to new colors, added toolbar to symptom report, updated symptom report to new UX design, updated list of symptoms in repo.

### CODE REVIEW

**Comment by mellowgeek** [comment]
Screenshot_1588472406

File: app/src/main/java/org/coepi/android/ui/thanks/ThanksViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+class ThanksViewModel(
+    private val navigation: RootNavigation
+) : ViewModel() {
+
+    fun onCheckInClick() {
+        navigation.navigate(ToDestination(actionGlobalSymptomsFragment()))
+    }
+
+    fun onSeeAlertsClick() {
+        navigation.navigate(ToDestination(actionGlobalAlerts()))
```
Comment (i-schuetz) [inline app/src/main/java/org/coepi/android/ui/thanks/ThanksViewModel.kt:19]: Is the navigation being handled correctly? This seems like it lets the thanks fragment and the form in the back stack, which is probably not wanted.

File: app/src/main/java/org/coepi/android/ui/symptoms/SymptomsViewModel.kt [hunk 2]
```diff
@@ -75,7 +78,7 @@ class SymptomsViewModel (
             .observeOn(mainThread())
             .subscribe{
                 uiNotifier.notify(it)
-                navigation.navigate(Back)
+                navigation.navigate(ToDestination(ThanksFragmentDirections.actionGlobalThanksFragment()))
```
Comment (i-schuetz) [inline app/src/main/java/org/coepi/android/ui/symptoms/SymptomsViewModel.kt:81]: Minor thing (but very quick to do), you can use static import to make this a little less verbose. If you press alt+enter on `actionGlobalThanksFragment` AS offers to add it.

File: app/src/main/res/layout/fragment_thanks.xml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            android:textColor="@color/white"
+            android:textSize="@dimen/text_size_headline"
+            android:textStyle="bold"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <Button
+            android:id="@+id/logmore"
+            style="@style/StandardButton"
+            android:layout_width="match_parent"
```
Comment (i-schuetz) [inline app/src/main/res/layout/fragment_thanks.xml:39]: `match_parent` doesn't work well in `ConstraintLayout`. It should be `0dp`.

File: app/src/main/res/layout/item_symptom.xml [hunk 4]
```diff
@@ -12,18 +12,22 @@
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="40dp"
-        android:gravity="center_vertical"
+        android:gravity="center_vertical|center_horizontal"
```
Comment (i-schuetz) [inline app/src/main/res/layout/item_symptom.xml:15]: Only `center`?

File: app/src/main/java/org/coepi/android/ui/thanks/ThanksViewModel.kt (hunk 1 above)
Comment (mellowgeek) [reply app/src/main/java/org/coepi/android/ui/thanks/ThanksViewModel.kt:19]: Adding `app:popUpTo="@id/tabs_container"` to each of those actions in nav_graph_root.xml fixes that behavior, so hitting the back button goes back to the home screen and never back to the thanks screen. Let me know if there's a better way of doing this.

Comment (i-schuetz) [reply app/src/main/java/org/coepi/android/ui/thanks/ThanksViewModel.kt:19]: Hmm, we need to remove the remaining tabs setup from the navigation. Let's ignore this for now. I'll do it later and fix this as part of it.

**Pull request merged**